x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/api/ml_models/fetch_model_api_logic.ts

```diff
@@ -0,0 +1,27 @@
+/*
```

Comment:
Not a new file, just moving and renaming `fetch_text_expansion_model_api_logic.ts`.